frontend/lib/parse-capi/index.ts

```diff
@@ -177,15 +178,11 @@ export const extractArticleMeta = (data: {}): CAPIType => {
         sectionName: getNonEmptyString(data, 'config.page.section'),
         pageId: getNonEmptyString(data, 'config.page.pageId'),
         sharingUrls: getSharingUrls(data),
+        pillar:
+            findPillar(getNonEmptyString(data, 'config.page.sectionName')) ||
+            'news',
```

Comment:
`sectionName != pillar`

For instance, [this article's](https://www.theguardian.com/fashion/2018/sep/27/which-female-celebrity-embodies-the-challenges-facing-women-today) `sectionName` is "Fashion"